### ISSUE TITLE
Option to reference sub rack files rather than copying the content

### ISSUE THREAD

**Issue by to7m** [falkTX/Carla#1354]
My use case is that I always have Carla running, and have several ‘sub racks’ (Carla Patchbay plugins) in this instance of Carla. To simplify things I'll say these sub racks are “Microphone Processing” and “Speakers Processing”, each containing an equaliser and volume adjustments. I have a different sub rack file for each Microphone Processing configuration, and a different sub rack file for each Speakers Processing configuration. I combine these into Carla files that function as presets, such as “laptop microphone and laptop speakers”, “laptop microphone and headphones”, “external microphone and headphones through Scarlett”, etc.

The issue is that if I have one session open, say “laptop microphone and headphones”, then I decide on an adjustment to the “laptop microphone” Microphone Processing sub rack, then I have to save that sub rack file, save the “laptop microphone and headphones” session file, then open all other affected session files (such as “laptop microphone and laptop speakers”) and reload the sub rack and save the session files.

To sum up, my suggestion is an option to specify whether a given sub rack is a reference to a sub rack file or not.
